v2/azure-arm.yaml

```diff
@@ -3710,6 +3710,8 @@ objectModelConfiguration:
       PublicIPPrefix:
         $export: true
         $supportedFrom: v2.1.0
+        $generatedConfigs:
```

Comment:
Dont use `$generatedConfigs`, we have CEL expressions now so this field isn't needed for users to export a plain status field.